#!/usr/bin/env python
PACKAGE = "sterela"
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

gen.add("right_dist", double_t, 0, "right_dist", 1.2 , 0, 10)

gen.add("arm_width", double_t, 0, "arm_width", 1.5 , 0, 10)

gen.add("left_dist", double_t, 0, "left_dist", 1.5 , 0, 10)

gen.add("max_range", double_t, 0, "max_range", 100 , 0, 100)

exit(gen.generate(PACKAGE, "sterela", "ObstacleDetector"))
    